pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[tool.poetry]
+name = "zoloto"
+version = "0.0.0"
+description = "A fiducial marker system powered by OpenCV - Supports ArUco and April"
+authors = ["Jake Howard <git@theorangeone.net>"]
+readme = "README.md"
+repository = "https://github.com/RealOrangeOne/zoloto"
+homepage = "https://github.com/RealOrangeOne/zoloto"
+
+[tool.poetry.dependencies]
+python = "^3.5"
+opencv-contrib-python-headless = "^4.1"
+cached-property = "^1.5"
+coordinates = "^0.3.0"
+fastcache = "^1.1"
+ujson = "^1.35"
+
+[tool.poetry.dev-dependencies]
+bandit = "^1.6"
+chrono = "^1.0"
+flake8 = "^3.7"
+flake8-commas = "^2.0"
+flake8-comprehensions = "^2.1"
+flake8-debugger = "^3.1"
+flake8-docstrings = "^1.3"
+flake8-isort = "^2.7"
+flake8-mutable = "^1.2"
+flake8-todo = "^0.7.0"
+flake8-print = "^3.1"
+flake8-tuple = "^0.4.0"
+hypothesis = "^4.28"
+mypy = "^0.720.0"
+pytest = "^5.0"
+pytest-benchmark = "^3.2"
+pytest-cov = "^2.7"
+ujson = "^1.35"
+isort = {version = "^4.3",extras = ["pyproject"]}
```

Comment:
What's the purpose of the `pyproject` extra?

Reply:
It adds a `toml` dependency so that isort can read `pyproject.toml`. See https://github.com/timothycrosley/isort/issues/705